packages/plugins/components/src/DefaultAppBarContent/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import classnames from 'classnames';
+import { FlyteLogo, useCommonStyles, headerFontFamily } from '@flyteconsole/ui-atoms';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { FlyteNavItem, NavigationDropdown } from '../NavigationDropdown';
+import { AppInfo, VersionInfo } from '../AppInfo';
+import { FeatureFlag, useFeatureFlag } from '../hooks';
+import { makeRoute } from '../Utils';
+import { OnlyMine } from '../OnlyMine';
+import { UserInformation } from '../UserInformation';
+
+
+const useStyles = makeStyles((theme: Theme) => ({
+  spacer: {
+    flexGrow: 1,
+  },
+  rightNavBarItem: {
+    marginLeft: theme.spacing(2),
+  },
+}));
+
+interface DefaultAppBarProps {
+  items: FlyteNavItem[];
+  console?: string;
+  versions: VersionInfo[];
+  routes: {
+    SelectProject: {
+      path: string
+    }
+  }
+}
+
+/** Renders the default content for the app bar, which is the logo and help links */
+export const DefaultAppBarContent = (props: DefaultAppBarProps) => {
+  const { console, items, versions, routes } = props;
```

Comment:
destructure props in definition?
```suggestion
export const DefaultAppBarContent = ({ console, items, versions, routes }: DefaultAppBarProps) => {
```